From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

bwren committed:
Logic App tutorial

bwren committed:
Logic App

bwren committed:
la-logic-app

bwren committed:
metadata

bwren committed:
metadata

bwren committed:
updates

bwren committed:
updates

bwren committed:
updates

bwren committed:
updates

bwren committed:
updates

bwren committed:
updates

bwren committed:
updates

bwren committed:
review version

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

LalithaMV committed:
CosmoDB: updated the author of this page

@mimig1

LalithaMV committed:
CosmosDB: Updating author

@mimig1

bwren committed:
last fixes, rename, toc

bwren committed:
toc

iainfoulds committed:
Renaming files

iainfoulds committed:
Adding redirects

iainfoulds committed:
Assigning ownership of docs to be retired

iainfoulds committed:
Retiring files

bwren committed:
Merge branch 'master' into la-logic-app

bwren committed:
title

bwren committed:
Merge branch 'la-logic-app' of https://github.com/bwren/azure-docs-pr into la-logic-app

t

iainfoulds committed:
Updating TOC

iainfoulds committed:
Setting docs to NOINDEX

bwren committed:
title

iainfoulds committed:
Fixing TOC entry

ShawnJackson committed:
edit pass: data-lake-analytics-data-lake-tools-debug-recurring-job

ShawnJackson committed:
edit pass: data-lake-analytics-data-lake-tools-debug-recurring-job

curtand committed:
updated note on changeover

curtand committed:
new links

curtand committed:
Acrolinx changes

curtand committed:
Note to Important

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cloudapp0102

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

Commit:
Specify code package when using open networking

Commit:
Specify using SSD for cluster data root

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cloudapp0102

curtand committed:
Sumi text

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

Duncanma committed:
Replacing iFrames with proper >[!VIDEO ] markdown extension

nitinme committed:
updating the support table

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

Duncanma committed:
Found some spelling/typo issues as I was looking into another issue

dennyglee committed:
Update to Spark Connector GA version

Commit:
Added tip box in Removing recommendations

Once index are discarded they will never be applied again by the system. This is a new behavior that is now updated in the docs.

davidbritch committed:
Update to reflect latest Firebase console.

davidbritch committed:
Updated to reflect latest Firebase console.

davidbritch committed:
Replaced GcmClient with Xamarin.Firebase.Messaging

Removed Xamarin component (GcmClient) and replaced it with Xamarin.Firebase.Messaging (NuGet package), along with links for more info.

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

tdykstra committed:
split input-output into separate sections

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

tamram committed:
updating owner and metadata

nitinme committed:
Changes for Jan 9

nitinme committed:
minor update

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Cosmos DB SQL API edits

curtand committed:
last classic portal fixes? i hope so.

HeidiSteen committed:
acro edits

MGoedtel committed:
added new troubleshooting article

MGoedtel committed:
fixed error and updated article metadata

MGoedtel committed:
updated to improve readabiility

mimig1 committed:
Cosmos DB: Updated endpoint for graph w/.net.

MGoedtel committed:
corrected link paths

MGoedtel committed:
corrected link paths

Commit:
Tutorial changes for furoture conversion to MVC.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas0108

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

daveba committed:
update saas tutorial list

iainfoulds committed:
Merge branch 'master' into vmclassicmoveswindows

v-geberr committed:
pricing tier

Commit:
Acrolinx fix.

v-geberr committed:
rework

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Mimi's edits

HeidiSteen committed:
one more sql api edit

Commit:
Bad link fix.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aas0108

iainfoulds committed:
Update connect-vms-classic.md

iainfoulds committed:
Update createportal-classic.md

v-geberr committed:
fixing note

v-geberr committed:
fixing h2

Commit:
Trying to get bad link error fixed.

cwatson-cat committed:
updated link to download publish cert file in Backup topic:

curtand committed:
date fix

cwatson-cat committed:
Updated download publish cert link in Cognitive Services topic:

v-geberr committed:
note

cwatson-cat committed:
Updated URLs for download publish cert in Gov dev guide

bwren committed:
images

MGoedtel committed:
updated metadata

daveba committed:
update saas tutorial list

KumudD committed:
addressed public PR 4051

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ILBupdate0108

v-geberr committed:
fixing links

curtand committed:
date fix

Commit:
adding Azure to H1

ktoliver committed:
Merge pull request #30553 from iainfoulds/vmclassicmoveswindows

VM Windows Classic doc retirements

cwatson-cat committed:
Fixed acrolinx score and info around download publish cert

rjagiewich committed:
Merge pull request #31066 from Minewiskan/aas0108

Aas0108

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

rjagiewich committed:
Merge pull request #31062 from MGoedtel/NewSupportArticle182018

added new troubleshooting article

ktoliver committed:
Merge pull request #25854 from bwren/la-logic-app

Log Analytics Logic App

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

ktoliver committed:
Merge pull request #31061 from curtand/classic0108

last classic portal fixes? i hope so.

nitinme committed:
splitting the support table

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

nitinme committed:
quick update

rjagiewich committed:
Merge pull request #31075 from cynthn/revision4

adding Azure to H1

ktoliver committed:
Merge pull request #31067 from MicrosoftDocs/saas-app-tutorial-list-update

update saas tutorial list

cynthn committed:
Update virtual-machines-common-accelerated-maintenance.md

rjagiewich committed:
Merge pull request #31074 from KumudD/ILBupdate0108

ILB text update

ktoliver committed:
Merge pull request #31068 from v-geberr/0108-rate-tiers

0108 rate tiers

rjagiewich committed:
Merge pull request #31072 from cwatson-cat/1-8-18-ClassicClnUP-UpdDwnld-PublishCertURL3

Updated URLs for download publish cert in Gov dev guide

ktoliver committed:
Merge pull request #31070 from cwatson-cat/1-8-18-ClassicClnUP-UpdDwnld-PublishCertURL

Updated link to download publish cert file in Backup topic

ktoliver committed:
Merge pull request #31071 from cwatson-cat/1-8-18-ClassicClnUP-UpdDwnld-PublishCertURL2

Updated download publish cert link in Cognitive Services topic:

cynthn committed:
Update virtual-machines-common-accelerated-maintenance.md

ktoliver committed:
Merge pull request #31078 from cynthn/patch-55

Update virtual-machines-common-accelerated-maintenance.md

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
Addressed Eugene's feedback: new H2 section for supported API types, deleted the known issue sentence, reverted 3 SQL API references back to AC DB as appropriate.

Commit:
Update iot-hub-arduino-iot-devkit-az3166-get-started.md

nitinme committed:
shuffling info between tables

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

GitHubber17 committed:
Merge pull request #31082 from shizn/master

Update iot-hub-arduino-iot-devkit-az3166-get-started.md

pharring committed:
Add more detail about Cloud Services temporary storage.

daveba committed:
update saas tutorial list

Duncanma committed:
Merge pull request #31021 from MicrosoftDocs/MoveVideosToMarkdown

Replacing iFrames with proper >[!VIDEO ] markdown extension

Duncanma committed:
Merge pull request #31086 from MicrosoftDocs/DuncanmaSpellingFixes

Found some spelling/typo issues as I was looking into another issue

linda33wj committed:
Tweak the connector topics

linda33wj committed:
Update compute LS topic

dennyglee committed:
Updated ms.date

nsoneji committed:
corrected typo

nsoneji committed:
minor correction.

Commit:
Update sql-database-advisor-portal.md

Commit:
Update sql-database-advisor-portal.md

Commit:
Remove region constraint for ASGs

Commit:
Removed location constraint.

pvrk committed:
Impact on backups by Azure Wide VM reboot

Recently we completed the forceful VM reboot across Azure and there has been an impact on Azure Backups

pvrk committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

rjagiewich committed:
Merge pull request #31085 from MicrosoftDocs/saas-app-tutorial-list-update

update saas tutorial list

GitHubber17 committed:
Merge pull request #31028 from danimir/patch-7

Added tip box in Removing recommendations

GitHubber17 committed:
Merge pull request #31033 from davidbritch/patch-9

Updated to reflect latest Firebase console.

rjagiewich committed:
Merge pull request #31032 from davidbritch/patch-8

Update to reflect latest Firebase console.

GitHubber17 committed:
Merge pull request #31035 from davidbritch/patch-10

Replaced GcmClient with Xamarin.Firebase.Messaging

GitHubber17 committed:
Merge pull request #31059 from HeidiSteen/heidist-master

Azure Search: Cosmos DB indexer edits

rjagiewich committed:
Update hdinsight-component-versioning.md

GitHubber17 committed:
Merge pull request #31064 from mimig1/01052018

Cosmos DB: Updated endpoint for graph w/.net.

cwatson-cat committed:
Updated cert publish URL in Backup topic

HeidiSteen committed:
fixed links to cosmosdb doc

GitHubber17 committed:
Merge pull request #31087 from linda33wj/master

Update ADF content

rjagiewich committed:
Merge pull request #31084 from pharring/master

Add more detail about Cloud Services temporary storage.

rjagiewich committed:
Merge pull request #31053 from nitinme/nitinme-freshness-updates

Updates to versioning article

GitHubber17 committed:
Merge pull request #31089 from nsoneji/master

corrected typo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

BryanLa committed:
Fix hyphen

GitHubber17 committed:
Merge pull request #31097 from jimdial/patch-54

Remove region constraint for ASGs

ArLucaID committed:
Fixed typo

GitHubber17 committed:
Merge pull request #31098 from jimdial/patch-55

Removed location constraint.

GitHubber17 committed:
Merge pull request #31102 from cwatson-cat/1-9-18-ClassicClnUP-Backup

Updated cert publish URL in Backup topic

BryanLa committed:
Fix hyphen

BryanLa committed:
Fix hyphen

rjagiewich committed:
Merge pull request #31041 from tdykstra/blob

split input-output into separate sections

Commit:
Merge pull request #30928 from mani-ramaswamy/master

Specify code package when using open networking

GitHubber17 committed:
Merge pull request #31103 from HeidiSteen/heidist-master

fixed links to cosmosdb doc

pvrk committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

GitHubber17 committed:
Merge pull request #31052 from tamram/tamram-0108

updating owner and metadata

PRMerger17 committed:
Merge pull request #31023 from dennyglee/master

Update to Spark Connector GA version

Ja-Dunn committed:
Merge pull request #30704 from ShawnJackson/data-lake-analytics-data-lake-tools-debug-recurring-job

edit pass: data-lake-analytics-data-lake-tools-debug-recurring-job

GitHubber17 committed:
Merge pull request #31106 from BryanLa/patch-16

Fix hyphen

GitHubber17 committed:
Merge pull request #31107 from arluca/patch-4

Fixed typo

GitHubber17 committed:
Merge pull request #31108 from BryanLa/patch-17

Fix hyphen

GitHubber17 committed:
Merge pull request #31109 from BryanLa/patch-18

Fix hyphen

GitHubber17 committed:
Merge pull request #31101 from pvrk/patch-30

Impact on backups by Azure Wide VM reboot

v-alje committed:
Merge pull request #31105 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger16 committed:
Merge pull request #30324 from LalithaMV/patch-2

CosmosDB: Updating author

PRMerger17 committed:
Merge pull request #30323 from LalithaMV/patch-1

CosmoDB: updated the author of this page

PRMerger18 committed:
Merge pull request #30728 from curtand/cloudapp0102

updated note on changeover

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo
